### ISSUE TITLE
Live TV and .strm files fail most of the time on 2.1.6

### ISSUE THREAD

**Issue by FreakyTurtle** [jellyfin-archive/jellyfin-roku-legacy#1970]
## Software Versions

- Jellyfin Server Version:  10.9.11
- Roku Client Version:  2.1.6

## Describe the bug
Since the new version of the roku client, playback fails without any ffmpeg logs, initially noticed on remote media sources (.strm files) managed to play one local file but not another, the majority of media is not playing, flashing an error message and either moving to the next episode, crashing out of the app, or crashing back to the home screen.  No ffmpeg logs are outputted.  The loading spinner never gets a percentage value.

## How To Reproduce

1. Go to Jellyfin on Roku v.2.1.6
2. Play any .strm media

## Expected behavior
Expect transcoding to begin, for ffmpeg logs to be available, and for previously working media to play correctly.

## Logs
[2024-10-08 12:51:53.834 +01:00] [INF] [32] Emby.Server.Implementations.HttpServer.WebSocketManager: WS "86.173.159.34" request
[2024-10-08 12:55:43.716 +01:00] [INF] [53] MediaBrowser.MediaEncoding.Encoder.MediaEncoder: Starting "/usr/lib/jellyfin-ffmpeg/ffprobe" with args "-analyzeduration 200M -probesize 1G -i \"[redacted url]" -threads 0 -v warning -print_format json -show_streams -show_chapters -show_format"
[2024-10-08 12:55:46.217 +01:00] [INF] [48] Jellyfin.Api.Helpers.MediaInfoHelper: User policy for "[redacted user]". EnablePlaybackRemuxing: True EnableVideoPlaybackTranscoding: True EnableAudioPlaybackTranscoding: True
[2024-10-08 12:55:46.994 +01:00] [INF] [55] Emby.Server.Implementations.Session.SessionManager: Playback stopped reported by app "Jellyfin Roku" "2.1.6" playing "All for One and One for Kim". Stopped at "0" ms
[2024-10-08 12:58:17.926 +01:00] [INF] [32] Emby.Server.Implementations.Session.SessionWebSocketListener: Sending ForceKeepAlive message to 1 inactive WebSockets.
[2024-10-08 13:01:05.925 +01:00] [INF] [65] Emby.Server.Implementations.Session.SessionWebSocketListener: Sending ForceKeepAlive message to 1 inactive WebSockets.

## Connection Information

- Is server local or remote? remote
- Is server connection HTTP or HTTPS? https

## Additional context
All the media plays fine in the web client, to try to deduce if there was an issue with certain types of media i got the playback info from the web client to see if there was an obvious cause

### Not working in Roku example 1
```
Playback Info
Player
Html Video Player
Play method
Direct streaming
Protocol
https
Stream type
HLS
Video Info
Player dimensions
2288x1226
Video resolution
1280x720
Dropped frames
0
Corrupted frames
0
Direct Streaming Info
Video codec
HEVC (direct)
Audio codec
AAC
Original Media Info
Container
mkv
Size
1.4 GiB
Bitrate
3.8 Mbps
Video codec
HEVC Main
Video bitrate
3.5 Mbps
Video range type
SDR
Audio codec
EAC3
Audio bitrate
256 kbps
Audio channels
6
Audio sample rate
48000 Hz
```
### Not working in roku example 2
```
Playback Info
Player
Html Video Player
Play method
Transcoding
Protocol
https
Stream type
HLS
Video Info
Player dimensions
2288x1226
Video resolution
1280x720
Dropped frames
0
Corrupted frames
0
Transcoding Info
Video codec
HEVC
Audio codec
AAC
Audio channels
2
Bitrate
10.0 Mbps
Transcoding progress
8.6%
Transcoding framerate
570 fps
Reason for transcoding
The container is not supported
Original Media Info
Container
mkv
Bitrate
192 kbps
Video codec
H264 High
Video range type
SDR
Audio codec
AAC LC
Audio bitrate
192 kbps
Audio channels
2
Audio sample rate
44100 Hz
```
### Working in roku example 1
```
Playback Info
Player
Html Video Player
Play method
Re-muxing
Protocol
https
Stream type
HLS
Video Info
Player dimensions
2288x1226
Video resolution
1920x1080
Dropped frames
0
Corrupted frames
0
Re-muxing Info
Video codec
HEVC (direct)
Audio codec
AAC (direct)
Original Media Info
Container
mkv
Size
905.9 MiB
Bitrate
3.0 Mbps
Video codec
HEVC Main
Video bitrate
2.9 Mbps
Video range type
SDR
Audio codec
AAC LC
Audio bitrate
127 kbps
Audio channels
2
Audio sample rate
48000 Hz
```

**Comment by am-periphery** [comment]
I have the same issue.
As a work-around I enabled Developer mode on the Roku (https://developer.roku.com/en-gb/docs/developer-program/getting-started/developer-setup.md) and then installed the v2.1.5 release (https://github.com/jellyfin/jellyfin-roku/releases/download/v2.1.5/Jellyfin-Roku-v2.1.5.zip) via the zip upload.

**Comment by FreakyTurtle** [comment]
I should also probably say I've tried it on 3 devices i own
Roku Streaming Stick (3810EU)
Roku Express 4K (3940EU)
Roku Express (3930EU)

**Comment by cewert** [comment]
Fixed by #1971 

2.1.7 coming asap thank you for the bug report
